wombat_ic_generation/generate_wombatlite_ic.py

```diff
@@ -0,0 +1,111 @@
+# Copyright 2023 ACCESS-NRI and contributors. See the top-level COPYRIGHT file for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# =========================================================================================
+# Generate constant initial conditions for WOMBATlite using grid from a provided file
+#
+# To run:
+#   python generate_wombatlite_ic.py --grid-filename=<path-to-grid-file>
+#       --variable-name=<variable-name> --ic-filename=<path-to-output-file>
+#
```

Comment:
Its not clear to me from this comment what type of grid file is expected, or what the variable name is for ? (Can there be a default variable name ?)

Reply:
This is described somewhat in the help output:

```bash
$ python generate_wombatlite_ic.py -h
usage: generate_wombatlite_ic.py [-h] --grid-filename GRID_FILENAME --variable-name VARIABLE_NAME --ic-filename
                                 IC_FILENAME

Generate constant initial conditions for WOMBATlite using grid from provided file.

options:
  -h, --help            show this help message and exit
  --grid-filename GRID_FILENAME
                        The path to a file containing a variable to copy the grid from.
  --variable-name VARIABLE_NAME
                        The name of the variable to copy the grid from.
  --ic-filename IC_FILENAME
                        The path to the initial condition file to be outputted.
```

Reply:
Resolved in [34b6caa](https://github.com/COSIMA/om3-scripts/pull/23/commits/34b6caa9d19840aa2c41720b93010db2b685dd75)